eth_utils/curried/__init__.py

```diff
@@ -37,12 +38,23 @@
     encode_hex,
     event_abi_to_log_topic,
     event_signature_to_log_topic,
+    filter_abi_by_name,
+    filter_abi_by_type,
     flatten_return,
     from_wei,
     function_abi_to_4byte_selector,
     function_signature_to_4byte_selector,
+    get_abi_input_names,
```

Comment:
It doesn't look like all of these are curried below. Should they be?

Reply:
There's a test which catches if any of the functions are missing. If I understand the test, some of the functions are auto-curried.

There is logic in https://github.com/ethereum/eth-utils/blob/78dc6459f5ea847d8e1a6b50bb656abadcdea865/tests/core/curried-utils/test_curried.py#L36 that determines whether functions should or should not be curried. Currying any of the other `ABI` functions makes the test fail.

Reply:
Ah, brilliant. Thanks for looking into it!